### DISCUSSION TITLE
Sensor stopped and can not start it again

### DISCUSSION THREAD

**Discussion by GeriKar** [NightscoutFoundation/xDrip#3024] [Bug reports] [answered]
Hello!  20 days ago I had a problem starting a sensor dexcom one with the new transmitters.  The phone is a Samsung Galaxy A13.  It started the sensor, after a while it unpair and when restarting the sensor it gave an error and didn't want to start it anymore.  After discussion in the xdrip group, we came to the decision that the transmitter is broken.  Tried it on other phone brands and nothing.  I bought a new transmitter and the sensor started up.  This lasted for 20 days, with quite frequent signal loss and transmitter disconnection.  That night it got unpaired and on pairing signal the sensor stopped and the same thing happened again like with the old transmitter.  All settings are made according to the model given in the xdrip group.  Is there anything to do or do I have to buy a new transmitter again?
![IMG-77066c34410c5024f84dfeb07396631c-V](https://github.com/NightscoutFoundation/xDrip/assets/38782755/dcaea9a3-3597-483b-a746-9085df1beeab)
![IMG-ef111b6ee00a1930c4db3c9cbf11c61c-V](https://github.com/NightscoutFoundation/xDrip/assets/38782755/977a6f94-31df-4bdb-8039-394880109bb9)
![IMG-144943cefd3d9e6fae3fa8e7b9f86020-V](https://github.com/NightscoutFoundation/xDrip/assets/38782755/ddf46226-a06d-43fe-99ad-e5bd0b285a3c)

**Comment by Navid200** [comment, marked as answer]
Why did xDrip prompt you, according to the logs, at 6:12 to approve a pair request?
Why did it unpair?

Can you post a screenshot of the following page?
`Settings` &#8722;> `G5/G6/Dex1 Debug Settings`  

Have you installed the last Samsung Android patch on your phone?  You must have got it after July 25.

**Reply by GeriKar** [reply]
![IMG-b2ed9859970da40e0c2b70e46e4e1854-V](https://github.com/NightscoutFoundation/xDrip/assets/38782755/d648a761-0099-4253-acbd-82629ec10c1d)
I do not know. I did not remember. From 3am to 6am i was trying to do something. Maybe i unpaired it manual.

**Reply by GeriKar** [reply]
On samsung a13 there is no update yet.

**Reply by Navid200** [reply]
Do you have a different phone, not Samsung, you can use until you get the update on Samsung?

**Reply by GeriKar** [reply]
No, unfortunately i do not have. But last time with other transmitter, i tried with xiaomi and did not worked. The issues was same.

**Reply by Navid200** [reply]
Xiaomi is not much better than Samsung.

**Reply by GeriKar** [reply]
That is a solution 😁 I take a new transmitter. I can try it and surrender 😀

**Reply by GeriKar** [reply]
I started the sensor with new transmitter and it started to count time. But after some minutes asked for pairing.
![IMG-ee2bc5b4954dd80557f9b8d10d48de5d-V](https://github.com/NightscoutFoundation/xDrip/assets/38782755/da680521-dc2d-4cdc-8fbf-9ca323c26da1)

**Reply by GeriKar** [reply]
I think it will start now but have i checked the special pairing to avoid unpairing?

**Reply by Navid200** [reply]
You started a new transmitter?  Why?  You cannot stop it.

**Reply by GeriKar** [reply]
Because with the old one i can not start the sensor.

**Reply by Navid200** [reply]
The problem is not starting sensor.  The problem is having proper connectivity with the transmitter.  The cause of the problem is your phone.

**Reply by GeriKar** [reply]
But i think i have a proper connectivity . Everything is like your example in troubleshooting.

**Reply by Navid200** [reply]
The probability of one person having three transmitters that all malfunction is Nil.

**Reply by GeriKar** [reply]
I am agree. But why with new transmitter, sensor starts. And what can i do? What ki d of phone to buy when xiaomi doesnt work well too

**Reply by Navid200** [reply]
Google Pixel is a solid phone for xDrip.
You can use the official Dexcom app if the phone is compatible.  If your phone is not compatible with the Dexcom app and you contact Dexcom, I don't think they can do anything about it.
